apps/web/src/features/layout/components/Header/header.module.css

```
@@ -160,15 +170,26 @@
 
 /* Light mode header styles - UPDATED FOR BETTER TEXT VISIBILITY */
 .header.lightMode {
-  background-color: rgba(252, 253, 255, 0.95); /* Higher opacity for better text contrast */
+  background-color: rgba(
+    252,
+    253,
+    255,
+    0.95
+  ); /* Higher opacity for better text contrast */
   border-bottom: 1px solid rgba(99, 123, 255, 0.12);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)